tests/src/cli.js

```diff
@@ -3,82 +3,107 @@
  */
 import path from 'path';
 
-import { expect } from 'chai';
-import { CLIEngine } from 'eslint';
+import { expect, use } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { CLIEngine, ESLint as ESLintClass } from 'eslint';
 import eslintPkg from 'eslint/package.json';
 import semver from 'semver';
+import * as importPlugin from '../../src/index';
+
+use(chaiAsPromised);
+
+const ESLint = ESLintClass || class {
+  constructor(options) {
+    options = Object.assign({},options);
+    const overrideConfig = options.overrideConfig;
+    delete options.overrideConfig;
+    const overrideConfigFile = options.overrideConfigFile;
+    delete options.overrideConfigFile;
+    const pluginsMap = options.plugins;
+    delete options.plugins;
+    this.engine = new CLIEngine(Object.assign(options, overrideConfig, overrideConfigFile ? { configFile:overrideConfigFile }: {}));
+
+    for (const [name, plugin] of Object.entries(pluginsMap || {})) {
+      this.engine.addPlugin(name, plugin);
+    }
+  }
+  lintFiles(params) {
+    const result = this.engine.executeOnFiles(params);
+    return Promise.resolve(result.results);
+  }
+};
```

Comment:
why do we need a hardcoded class here? is this no longer exposed, even under legacy specifiers?

Reply:
This plugin also supports ESLint <= v6, so I created a compatible class to see if it works for testing ESLint <= v6.

Reply:
I'd prefer not to use such a wrapper if possible; that would mean we're not directly testing eslint < 7, and our wrapper could hide bugs.

Reply:
I made change to this PR.
https://github.com/import-js/eslint-plugin-import/pull/2191/commits/184118366b6022dda011fa8eb170b598a6738b50